packages/block-library/src/rss/block.json

```diff
@@ -46,6 +46,22 @@
 		"html": false,
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true
```

Comment:
Theme.json and user global styles are being overwritten on the frontend by the RSS block's styles:

https://github.com/WordPress/gutenberg/blob/11696c16909453af1b4f9d015ace24b7ee7c678b/packages/block-library/src/rss/style.scss#L2-L3

It's there to remove the default list padding.

Here's the PR that introduced it: https://github.com/WordPress/gutenberg/pull/33294

Reducing specificity might work:

```css
:where(ul.wp-block-rss) {
	padding: 0;
}
```

Reply:
For clarity, I believe the suggestion here is to extract the padding to a new style with the reduced specificity, as shown above, rather than change the selector of the entire block of styles.

Reply:
Exactly, thanks for translating my fly-by comment 👍🏻 😄

Reply:
Extracted the padding to a new style in the recent commit.

Reply:
Unfortunately, the original padding style wasn't removed so the problem still exists on the frontend.